apps/wallet-mobile/src/features/Swap/useCases/CreateOrder/StartSwapOrderScreen.tsx

```diff
@@ -0,0 +1,223 @@
+import {useTheme} from '@yoroi/theme'
+import * as React from 'react'
+import {StyleSheet, Text, useWindowDimensions, View} from 'react-native'
+import {ScrollView} from 'react-native-gesture-handler'
+
+import {Button, ButtonType} from '../../../../components/Button/Button'
+import {Icon} from '../../../../components/Icon'
+import {useModal} from '../../../../components/Modal/ModalContext'
+import {RefreshButton} from '../../../../components/RefreshButton/RefreshButton'
+import {Space} from '../../../../components/Space/Space'
+import {useIsKeyboardOpen} from '../../../../kernel/keyboard/useIsKeyboardOpen'
+import {usePortfolioBalances} from '../../../Portfolio/common/hooks/usePortfolioBalances'
+import {useSelectedWallet} from '../../../WalletManager/common/hooks/useSelectedWallet'
+import {AmountCard} from '../../common/AmountCard/AmountCard'
+import {useNavigateTo} from '../../common/navigation'
+import {useStrings} from '../../common/strings'
+import {useSwap} from '../../common/SwapProvider'
+import {EditPrice} from './EditPrice'
+import {ProviderSplits} from './ProviderSplits'
+
+// const LIMIT_PRICE_WARNING_THRESHOLD = 0.1 // 10%
```

Comment:
does it need to be implemented?

Reply:
I think yes, could double check with product but I believe we used to have a warning if the limit price was too different than the market one. I'm guessing we'll keep that.